lib/dialects/mssql/query-generator.js

```diff
@@ -685,6 +685,11 @@ var QueryGenerator = {
     if (options.limit || options.offset) {
       if (!options.order || (options.include && !orders.subQueryOrder.length)) {
         fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
+
+        if(model.name){
+          fragment += this.quoteIdentifier(model.name) + '.';
+        }
```

Comment:
On vacation where I only get signal when I come down the Smokey Mountains of North Carolina. I will change my code to fit your specifications in 5 days